packages/cli/src/tests/synthql.dev.config.json

```diff
@@ -0,0 +1,179 @@
+{
```

Comment:
what is the purpose of this file?

Reply:
nvm, for the tests

Reply:
It's a copy of the config schema I'm using to validate the version I'm using for the tests

Reply:
Can you add a comment somewhere (i.e. where you generate it) explaining what this is for